x-pack/platform/plugins/shared/fleet/public/applications/integrations/sections/epm/screens/detail/settings/rollback_button.tsx

```diff
@@ -96,6 +98,11 @@ export function RollbackButton({ packageInfo, isCustomPackage }: RollbackButtonP
                 id="xpack.fleet.integrations.rollbackPackage.licenseTooltip"
                 defaultMessage="Rollback integrations requires an enterprise license."
```

Comment:
```suggestion
                defaultMessage="Rolling back integrations requires an Enterprise license."
```
I think that text was incorrect. Put this way it sounds like there is an integration named "Rollback". And Enterprise needs to be capitalized. I know it's untouched by this PR, I just happened to read it :)